Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -72,7 +72,8 @@ class Navigator {
                                                      config: NavigatorConfig(),
                                                      customConfig: "")
         
-        instance.historyRecorder = try! HistoryRecorderHandle.build(forConfig: configFactory)
+        instance.historyRecorder = try! HistoryRecorderHandle.build(forHistoryFile: "",
```

Comment:
Shall we provide additional parameter in shared navigator to provide custom history file path?

Reply:
Where does the history file go by default? This is a shared object without a public initializer, so the only opportunity to configure this option would be in Info.plist. Otherwise, if we can choose a decent default location, that could work for now.